python/pyspark/conf.py

```diff
@@ -60,11 +62,11 @@ class SparkConf:
 
     Examples
     --------
-    >>> from pyspark.core.conf import SparkConf
-    >>> from pyspark.core.context import SparkContext
+    >>> from pyspark.conf import SparkConf
+    >>> from pyspark.context import SparkContext
```

Comment:
The PR title doesn't mention `SparkContext` change at all. Could you proceed `SparkContext` change as a separate PR at least please, @HyukjinKwon ?
```
Make SparkConf to root level to for both SparkSession and SparkContext
```

Reply:
The example does not need to be changed actually. I just changed to use the API instead of internal package `pyspark.context`.

Just to be clear:

- `from pyspark.core.context import SparkContext`
- `from pyspark.context import SparkContext`
- `from pyspark.import SparkContext`

all work fine. There's no behaviour change

Reply:
Ah, got it~